stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/adapter/BaseMessageItemViewHolder.kt

```diff
@@ -9,8 +9,10 @@ public abstract class BaseMessageItemViewHolder<T : MessageListItem>(
     itemView: View,
     private val decorators: List<Decorator>
 ) : RecyclerView.ViewHolder(itemView) {
+    protected lateinit var data: T
```

Comment:
Could use `private set` to be more precise, and read-only to implementations

Also though, thanks for fixing this 👍